graphql/testdata/task_logs-event_logs-success.graphql

```diff
@@ -0,0 +1,18 @@
+query {
+  taskLogs(
+    taskId: "logkeeper_ubuntu_test_edd78c1d581bf757a880777b00685321685a8e67_16_10_20_21_58_58"
+  ) {
+    eventLogs {
+      timestamp
+      eventType
+      data {
+        hostId
+        jiraIssue
+        priority
+        status
+        timestamp
+        userId
+      }
```

Comment:
we should stop contributing to old tests and from hereon put all tests in new testing method.